unit_tests/test_nemesis.py

```diff
@@ -223,3 +226,124 @@ def test_use_disabled_monkey():
     collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
 
     assert 'disrupt_toggle_table_gc_mode' in collected_disrupt_methods_names
+
+
+class TestSisyphusMonkeyNemesisFilter:
+    # pylint: disable=no-self-use
+    @pytest.fixture(autouse=True)
+    def expected_topology_changes_methods(self):
+        return [
+            "disrupt_restart_with_resharding",
+            "disrupt_nodetool_seed_decommission",
+            "disrupt_nodetool_drain",
+            "disrupt_nodetool_decommission",
+            "disrupt_add_remove_dc",
+            "disrupt_grow_shrink_cluster",
+            "disrupt_terminate_and_replace_node",
+            "disrupt_decommission_streaming_err",
+            "disrupt_remove_node_then_add_node"]
+
+    @pytest.fixture(autouse=True)
+    def expected_schema_changes_methods(self):
+        return [
+            "disrupt_add_drop_column",
+            "disrupt_toggle_table_ics",
+            "disrupt_toggle_cdc_feature_properties_on_table",
+            "disrupt_modify_table",
+            "disrupt_toggle_table_gc_mode",
+        ]
+
+    @pytest.fixture(autouse=True)
+    def expected_config_changes_methods(self):
+        return [
+            "disrupt_modify_table",
+            "disrupt_toggle_cdc_feature_properties_on_table",
+            "disrupt_hot_reloading_internode_certificate",
+            "disrupt_grow_shrink_new_rack",
+            "disrupt_restart_with_resharding",
+            "disrupt_rolling_config_change_internode_compression",
+            "disrupt_rolling_restart_cluster",
+            "disrupt_switch_between_password_authenticator_and_saslauthd_authenticator_and_back",
+            "disrupt_resetlocalschema",
+        ]
+
+    @pytest.fixture(autouse=True)
+    def expected_config_and_schema_changes_methods(self):
+        return [
+            "disrupt_toggle_cdc_feature_properties_on_table",
+        ]
+
+    def test_list_topology_changes_monkey(self, expected_topology_changes_methods):
```

Comment:
test with such name exists (line 175) - and looks similar.

Reply:
Yes, it was duplicated. Removed one